index.html

```diff
@@ -1,41 +1,191 @@
 <!DOCTYPE html>
-
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:wght@100&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
     <script src="https://kit.fontawesome.com/d55257b723.js" crossorigin="anonymous"></script>
+    <title>My Portfolio</title>
 </head>
 <body>
-    <header class="header">
+    <header>
         <a href="#" class="logo">Nonhlanhla</a>
-        <nav><a href="#"><i class="fas fa-bars"></i></a></nav>
-        
-
-    
-</header>
-<section>
-<div class="section">
-    <div class="width"></div>
-    <div class="Home-content">
-        <h1>Hey There.<br>I'm Nonhlanhla Mndebele</h1>
-        <h2>I'am a Software Developer</h2>
-        <p>I can help you build a product, feature or website look
-            through some of my work and experience! If you like
-            what you see and have a project you need coded,
-            don't hesitate to contact me.</p>  
-        <ul class="social">
-            <li class="github a"><a href="ct"><img src="/assets/icons/git.png" alt="img"></a></li>
-            <li class="linkedin a"><a href="ct"><img src="assets/icons/i.png" alt="img"><span><img src="assets/icons/n.png" alt="img"></span></a></li>
-            <li class="angellist a"><a href="ct"><img src="assets/icons/two.png" alt="img"></a></li>
-            <li class="twitter a"><a href="ct"><img src="assets/icons/twet.png" alt="img"></a></li>
-            <li class="medium a"><a href="ct"><img src="assets/icons/med.png" alt="img"></a></li>
-        </ul>
-    </div>
-</div> 
-</section>
+        <nav>
+            <a href="#">
+                <i class="fas fa-bars"></i>
+            </a>
+        </nav>
+    </header>
+    <section>
+        <div class="home-content">
+            <h1>Hey There. <br>I’m Nonhlanhla Mndebele </h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product, feature or website look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <ul class="social">
+                <li class="github-a">
+                    <a href="#">
+                        <img src="/assets/icons/git.png" alt="img">
```

Comment:
All images are not loading, please correct it by adding a `.` before `/assets`

` 
<img src="/assets/icons/git.png" alt="img">
`
Add '.'
` 
<img src="./assets/icons/git.png" alt="img">
`

<img width="441" alt="image" src="https://user-images.githubusercontent.com/7243132/170840598-643280d6-d3d6-4500-924f-6cb5cd7b98fd.png">